pkg/cli/image_cmd.go

```diff
@@ -129,11 +129,12 @@ func setupImageFlags(imageCmd *cobra.Command, searchImageParams map[string]*stri
 	searchImageParams["baseImage"] = imageCmd.Flags().StringP("base-images", "b", "",
 		"List images that are base for the given image")
 
-	imageCmd.Flags().StringVar(servURL, "url", "", "Specify zot server URL if config-name is not mentioned")
-	imageCmd.Flags().StringVarP(user, "user", "u", "", `User Credentials of zot server in "username:password" format`)
-	imageCmd.Flags().StringVarP(outputFormat, "output", "o", "", "Specify output format [text/json/yaml]")
-	imageCmd.Flags().BoolVar(verbose, "verbose", false, "Show verbose output")
-	imageCmd.Flags().BoolVar(debug, "debug", false, "Show debug output")
+	imageCmd.PersistentFlags().StringVar(servURL, "url", "", "Specify zot server URL if config-name is not mentioned")
+	imageCmd.PersistentFlags().StringVarP(user, "user", "u", "",
+		`User Credentials of zot server in "username:password" format`)
+	imageCmd.PersistentFlags().StringVarP(outputFormat, "output", "o", "", "Specify output format [text/json/yaml]")
+	imageCmd.PersistentFlags().BoolVar(verbose, "verbose", false, "Show verbose output")
+	imageCmd.PersistentFlags().BoolVar(debug, "debug", false, "Show debug output")
 }
 
 func searchImage(searchConfig searchConfig) error {
```

Comment:
is this still used?

Reply:
Yes they are used by the command, we are keeping it until removed